humatest/humatest.go

```diff
@@ -16,22 +16,23 @@ import (
 // NewRouter creates a new test router. It includes a logger attached to the
 // test so if it fails you will see additional output. There is no recovery
 // middleware so panics will get caught by the test runner.
```

Comment:
I actually find that very annoying, because I _think_ that if a single test fails you'll get log spew from all the tests (making debugging harder). This would be even worse for table-drive tests where maybe just one case is failing. I would like to be able to at least have the option to not attach a logger.

Reply:
You have a couple of options for this:

1. Call `humatest.NewRouter` for each test in the table, then you get only the single failing log output.
1. Ignore the `humatest` package and just do `huma.New` instead and you get a blank router without any middleware.